data-reconciliation-app/Makefile

```diff
@@ -10,20 +10,29 @@ build: ## 🔨 Build the Application
 	@echo -e "\e[34m$@\e[0m" || true
 	@npm run build
 
-build-virtual: build ## 📦 Build Virtual container image from Dockerfile
+run-env-setup: create-jwt-configs build  ## 🔨 Create the initialization configs and Build the Application
+	@echo -e "\e[34m$@\e[0m" || true
+
+build-virtual: run-env-setup ## 📦 Build Virtual container image from Dockerfile
 	@echo -e "\e[34m$@\e[0m" || true
 	@../scripts/build_image.sh virtual
 
-build-enclave: build ## 📦 Build Enclave container image from Dockerfile
+build-enclave: run-env-setup ## 📦 Build Enclave container image from Dockerfile
 	@echo -e "\e[34m$@\e[0m" || true
 	@../scripts/build_image.sh enclave
 
-# Start hosting the application using `sandbox.sh` 
-start-host: build ## 🏃 Start the CCF network using Sandbox.sh
+# Start hosting the application using `sandbox.sh` and enable custom JWT authentication
+start-host: run-env-setup  ## 🏃 Start the CCF network using Sandbox.sh
```

Comment:
Should both of these host's use the jwt issuer? If they do then what is the difference between them? (It might be a git merge thing going on here!)

Reply:
One is using MS-AAD (as an identity provider), and the other is using Test-IdP.